environment.d.ts

```diff
@@ -1,11 +1,11 @@
 declare namespace NodeJS {
   interface ProcessEnv {
-    NAVIGATION_MENU_HOME_URL: string
-    NAVIGATION_MENU_CREATE_URL: string
-    NAVIGATION_MENU_EXPLORE_URL: string
-    NAVIGATION_MENU_INTERACT_URL: string
-    NAVIGATION_MENU_LEARN_URL: string
-    NAVIGATION_MENU_OKP4_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_HOME_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_CREATE_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_EXPLORE_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_INTERACT_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_LEARN_URL: string
+    NEXT_PUBLIC_NAVIGATION_MENU_OKP4_URL: string
     OKP4_DISCORD_URL: string
```

Comment:
The social media urls + website do not work, they need to be made public like the navigation variables. I have not tested the other variables